Stack/Opc.Ua.Core/Types/Encoders/JsonDecoder.cs

```diff
@@ -1115,13 +1038,39 @@ public ExpandedNodeId ReadExpandedNodeId(string fieldName)
             }
         }
 
-
         /// <summary>
         /// Reads an StatusCode from the stream.
         /// </summary>
         public StatusCode ReadStatusCode(string fieldName)
         {
-            return ReadUInt32(fieldName);
+            object token;
+            // read non reversible status code
```

Comment:
are here and below handling the non reversible case?

Reply:
it has handling for non reversible case
but the comment was duplicated. the logic for non-reversible handling is a few lines below. fixed now..
```
                // try to read the non reversible Code
                if (ReadField("Code", out token))
```